halpybot/packages/configmanager/config.py

```diff
@@ -37,11 +33,4 @@ def config_write(module: str, key: str, value):
         value (str): New value
 
     """
-    logger.info("{module}, {key} set to {value}", module=module, key=key, value=value)
-    config[module][key] = value
-    try:
-        with open("config/config.ini", "w", encoding="UTF-8") as conf:
-            config.write(conf)
-    except (FileNotFoundError, PermissionError) as ex:
-        logger.exception("Error writing value to config file. Check permissions?")
-        raise ConfigException from ex
+    raise NotImplementedError("# TODO")  # TODO
```

Comment:
Noting TODO for reference.

Reply:
I believe we agreed to remove this function entirely.
Do we want to do that in this PR or in a follow up?

Reply:
Explosive removal approved.

Reply:
Charges set.